code/game/machinery/status_display.dm

```diff
@@ -32,6 +36,51 @@
 	var/index1 // display index for scrolling messages or 0 if non-scrolling
 	var/index2
 
+/obj/item/wallframe/status_display
+	name = "status display frame"
+	desc = "Used to build status displays, just secure to the wall."
+	icon_state = "unanchoredstatusdisplay"
+	custom_materials = list(/datum/material/iron=14000, /datum/material/glass=8000)
+	result_path = /obj/machinery/status_display
+
+//Copy pasted decon paths from newscaster.dm
+/obj/machinery/status_display/wrench_act_secondary(mob/living/user, obj/item/tool)
+	. = ..()
+	user.balloon_alert(user, "[anchored ? "un" : ""]securing...")
+	tool.play_tool_sound(src)
+	if(tool.use_tool(src, user, 6 SECONDS))
+		playsound(loc, 'sound/items/deconstruct.ogg', 50, TRUE)
+		user.balloon_alert(user, "[anchored ? "un" : ""]secured...")
+		deconstruct()
+		return TRUE
+
+/obj/machinery/status_display/welder_act(mob/living/user, obj/item/tool)
+	if(machine_stat & BROKEN)
+		if(!tool.tool_start_check(user, amount=0))
+			return
+		user.visible_message(span_notice("[user] is repairing [src]."), \
+							span_notice("You begin repairing [src]..."), \
+							span_hear("You hear welding."))
```

Comment:
Use balloon alerts instead.

Reply:
In this case, just "repairing..."

Reply:
remove the span_hear and the second span_notice?